Python/rerf/rerfClassifier.py

```diff
@@ -168,23 +202,62 @@ def fit(self, X, y):
             # [:, np.newaxis] that does not.
             y = np.reshape(y, (-1, 1))
 
+        num_obs = len(y)
+
         # setup the forest's parameters
         self.forest_ = pyfp.fpForest()
 
-        if self.projection_matrix == "RerF":
```

Comment:
well.  here it is. reason why OOB stuff isn't working.  your s-rerf PR conflicted. and neither of us caught it.  and i didn't have any tests for oob so travis didn't fail.